.gitignore

```diff
@@ -52,22 +52,22 @@ dist/
 compile_commands.json
 
 # Example resources
-examples/test_data/open3d_downloads
+data/common/downloads
```

Comment:
`download` instead of `downloads`, see: https://github.com/isl-org/Open3D/tree/yixing/data-dir-mockup/data